com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/ShaderPass.template

```diff
@@ -25,6 +25,7 @@ Pass
     $splice(GraphKeywords)
 
     #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+    #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Texture.hlsl"
```

Comment:
arf, we noticed an issue (haven't see it in the 10.x Pr), the Texture.hlsl MUST be define after RaytracingMacro.hlsl in our case as this file override the SAMPLE_TEXTURE2D_LOD / SAMPLE_TEXTURE2D Macro to work correctly with raytrace.

Reply:
Verified it does the correct thing -- because the SampleTexture* nodes directly call the SAMPLE_TEXTURE2D() macros on the hlsl Texture2D (not using the functions defined in Texture.hlsl), it will call the raytracing-defined macro.

Even if a user has a custom function node that is manually calling SAMPLE_TEXTURE2D(UnityTexture2D) it will still continue to work, as when raytracing redefines the macro, it will switch from calling UnityTexture2D.Sample() to calling UnityTexture2D.SampleLevel(), which works in raytracing.